src/atoms/tile/AppTile.ts

```diff
@@ -0,0 +1,38 @@
+import { TTile } from '../../types';
+
+export function AppTile({
+    height,
+    width = 'w-full',
+    textContent,
+    textSize,
+    imgUrl,
+    imgOptions,
+    onClick,
+}: TTile): HTMLElement {
+    const tile = document.createElement('div');
+    tile.className = `rounded-lg bg-tile-bg-gray flex items-center`;
+    tile.classList.add(width, height);
+
+    if (imgUrl) {
+        const img = document.createElement('img');
+        if (imgOptions) {
+            img.classList.add(...imgOptions);
+        }
+        img.src = imgUrl;
```

Comment:
minor suggestion: to add **img.alt** also

Reply:
Added